BRANCHES.md

```diff
@@ -0,0 +1,46 @@
+# Maintained branches
+
+At any point in time, we have a number of maintained branches consisting of:
+
+- the development branch: this is where new features lands, as well as bug
+  fixes and security fixes
+- one or more LTS branches: these only get bug fixes and security fixes.
+
+We use [Semantic Versioning](https://semver.org/). In particular, we maintain
+API compatibility in the development branch between major version changes. We
```

Comment:
We don't actually maintain API compatibility in `development`. We do in `master`. It happens from time to time that we tweak a new feature that just landed into `development` if it hasn't been part of a release yet.